roles/update/templates/monitor_resources_changes.sh.j2

```diff
@@ -0,0 +1,278 @@
+#!/bin/bash
+#
+# Description: The script monitors pods in openstack and
+# openstack-operator namespaces. It also monitors the containers and
+# rpm on the compute hosts.  If a change happens, it logs a diff style
+# of the change that happened into TIMELINE_LOG_FILE.
+#
+# Unless called with `-a init` or `-f` this script will daemonize
+# itself.
+#
+set -euo pipefail
+
+# Initialize default options
+SLOG_FILE="/dev/null"           # Script log file
+FOREGROUND=false
+ACTION="all" # default action
+TIME_UNIT=10 # default time unit
+BASE_DIR="{{ cifmw_update_artifacts_basedir }}"
+TIMELINE_LOG_FILE="${BASE_DIR}/update_timeline.log"
+
+# Where to find the inventory to connect to the compute.
+CI_INVENTORY="{{ ansible_user_dir }}/ci-framework-data/artifacts/zuul_inventory.yml"
+
+# Log files
+UPDATE_STAGE_FILE="${BASE_DIR}/current_update_stage.log"
+
+# OpenShift variables
+OS_NAMESPACES=("openstack-operators" "openstack")
```

Comment:
Use variables from roles

Reply:
hum ... so we have `cifmw_update_namespace` as a var in the update role ([there](https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/update/defaults/main.yml#L33) but we don't have a local to the role `openstack-operators` variable in the role (unless I missed it)

At the cifmw level the `openstack-operators` namespace name seems pretty much hardcoded.
So, given that `update` doesn't need a specific namespace, I'm not sure we should have any variable for namespaces as `openstack` and `openstack-operators` are pretty much hardcoded right now.

Given that having multiple namespace for `openstack` is coming (is there ?) that still might be something we need to do, but here, not sure.